src/components/Header/Header.js

```diff
@@ -1,10 +1,55 @@
-import React from 'react';
+import { Component } from 'react';
+import { withRouter } from 'react-router-dom';
 import './Header.scss';
 
-class Header extends React.Component {
+export class Header extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isActivate: false,
+    };
+  }
+  componentDidMount() {
+    window.addEventListener('scroll', this.handleScroll);
+  }
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.handleScroll);
+  }
```

Comment:
메서드 사용시에도 줄바꿈 해서 가독성을 높여주세요!
```suggestion
  constructor() {
    super();
    this.state = {
      isActivate: false,
    };
  }
  
  componentDidMount() {
    window.addEventListener('scroll', this.handleScroll);
  }
  
  componentWillUnmount() {
    window.removeEventListener('scroll', this.handleScroll);
  }
```

Reply:
도현님 항상 주의하며 리팩토링 하겠습니다 감사합니다!!